bin/docker-entrypoint

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
+
```

Comment:
do we need to add this jemalloc code?
https://github.com/rails/rails/blob/060d6ca4d111d1e9b14a0a2aa4d3da12718fcb1a/railties/lib/rails/generators/rails/app/templates/docker-entrypoint.tt#L3-L6

Reply:
We are already doing it in the Dockerfile: https://github.com/rootstrap/rails_api_base/pull/732/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R75

I think it makes more sense to have it as a `RUN` step (build time) than moving it to the `ENTRYPOINT` command (Run time). Just checked and the env variable is correctly setup on runtime
![Uploading image.png…]()

Reply:
ahhh ok great. Maybe you have a PR for Rails then 😉